utils/src/lib.rs

```diff
@@ -78,12 +78,17 @@ pub fn le_bytes_to_bigint<const BYTES_SIZE: usize, const NUM_LIMBS: usize>(
 /// assert_eq!(truncated, [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17,
 ///                        18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32]);
 /// ```
+// (Jorrit) this is confusing because the value returned is 32 bytes but the actual values is 31 bytes
 pub fn truncate_to_circuit(bytes: &[u8; 32]) -> [u8; 32] {
```

Comment:
My bad, we should change the return type to `[u8; 31]`.

Reply:
Let's hold off for now though I want to change the hash and truncating scheme to do without truncating similar to how pdas are created with a bump so that we don't need to truncate at all.